src/main/java/com/google/devtools/build/lib/remote/util/TracingMetadataUtils.java

```diff
@@ -52,6 +51,24 @@ public static RequestMetadata buildMetadata(
       String commandId,
       String actionId,
       @Nullable ActionExecutionMetadata actionMetadata) {
+    return buildMetadata(
+        buildRequestId,
+        commandId,
+        actionId,
+        actionMetadata != null ? actionMetadata.getProgressMessage() : null,
```

Comment:
Is there a specific reason why we use `actionMetadata.getProgressMessage()` instead of `actionMetadata.getMnemonic()`?

I am trying to change the type of `actionMetadata` to `ActionAnalysisMetadata` in order to make import of https://github.com/bazelbuild/bazel/pull/25448 easier.

Reply:
It's actually a bug, could you change it back?

Reply:
@iancha1992 Could you track this as a blocker for 8.2.0?